### PR TITLE
Suggestions for PR #930

### PR DESCRIPTION
**Pull request by mloning**
#### Reference Issues/PRs
PR #930

#### What does this implement/fix? Explain your changes.
* file name changes
* minor fixes

### CODE REVIEW

**Comment by mloning** [comment]
Hi @afzal442 I'm a bit stuck and out of my depth here with HTML, I can't manage to create the table locally. Could you ping me on slack to set up a call?

**Comment by fkiraly** [comment]
already suggested that in #930, would like to be part of that call.

**Comment by afzal442** [comment]
Thanks, sure. I am happy to chat. I left some comments, plz also have a look.

File: docs/source/estimator_overview.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.. _estimator_overview:
+
+==================
+Estimator Overview
+==================
+
+The table below gives an overview of all estimators in sktime.
+
+.. raw:: html
+   :file: estimator_overview/estimator_overview.html
```
Comment (afzal442) [inline docs/source/estimator_overview.rst:10]:
```suggestion
   :file: ../../estimator_overview.html
```

**Review by afzal442 — CHANGES_REQUESTED** [review]

File: docs/source/estimator_overview/estimator_overview.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  </style>
+</head>
+
+<body>
+<label for="myInput"></label><input type="text" id="myInput" placeholder="Search for names..">
+  <br>
+  <br>
+
+  <div
+          w3-include-html="estimator_overview_table.html"></div>
```
Comment (afzal442) [inline docs/source/estimator_overview/estimator_overview.html:77]:
```suggestion
          w3-include-html="../source/estimator_overview/estimator_overview_table.html"></div>
```

**Review by afzal442 — CHANGES_REQUESTED** [review]

File: docs/source/estimator_overview/estimator_overview.html (hunk 2 above)
Comment (afzal442) [reply docs/source/estimator_overview/estimator_overview.html:77]: Hopefully, it'll work.
```suggestion
          w3-include-html="estimator_overview_table.html"></div>
```

Comment (mloning) [reply docs/source/estimator_overview/estimator_overview.html:77]: Is it working for you locally? If so could you tell me, what you run in what sequence?

Comment (afzal442) [reply docs/source/estimator_overview/estimator_overview.html:77]: Yes, it's working now fine. But it'll run through estimator_overview.rst file only.

Comment (mloning) [reply docs/source/estimator_overview/estimator_overview.html:77]: Okay, so what do you run to generate the docs?

Comment (afzal442) [reply docs/source/estimator_overview/estimator_overview.html:77]: once you run a command using sphinx. Thanks.

Comment (afzal442) [reply docs/source/estimator_overview/estimator_overview.html:77]: I guess `make docs` but sorry, I ran `sphinx-build source built`.

Comment (mloning) [reply docs/source/estimator_overview/estimator_overview.html:77]: When you run this, `open built/index.html` and navigate to the estimator overview, do you see the table? I still can't see it ... 🤔

Comment (afzal442) [reply docs/source/estimator_overview/estimator_overview.html:77]: Thanks Markus. I see the table when I run that. Let's see at the meeting soon.

**Pull request closed without merging**